_site/blog/welcome-to-jekyll/index.html

```diff
@@ -70,19 +73,19 @@ <h1>Welcome to Jekyll!</h1>
   </div>
   <div class="grid__item width-4-12 width-12-12-m"><div class="share-page">
   <a class="share-linkedin" href="https://www.linkedin.com/shareArticle?mini=true&url=http://localhost:4000&title=" rel="nofollow" target="_blank" title="Share on LinkedIn">
-    <img src="//assets/images/share-page/icons_social-linkedin.png"/>
+    <img src="/assets/images/share-page/icons_social-linkedin.png"/>
   </a>
-  <a class="share-twitter" href="https://twitter.com/intent/tweet?text=&url=http://localhost:4000&via=project_twitter_username&related=project_twitter_username" rel="nofollow" target="_blank" title="Share on Twitter">
-    <img src="//assets/images/share-page/icons_social-twitter.png"/>
+  <a class="share-twitter" href="https://twitter.com/intent/tweet?text=&url=http://localhost:4000&via=kogito_kie&related=kogito_kie" rel="nofollow" target="_blank" title="Share on Twitter">
```

Comment:
is this url correct?
localhost:4000

Reply:
This is generated content. I changed the twitter account to kogito_kie because I thought it might be used somewhere .. well, we should use twitter link in the web somewhere but I would add some icon / feed in some other PR.

Reply:
@ibek can this url value be changed to _https://kogito.kie.org/_. Otherwise, this localhost is shown in the tweet text.

Reply:
This is some welcome to jekyll page ... it's not used anywhere at https://kogito.kie.org AFAIK. I can remove the page completely or as I said I can add the share twitter button somewhere to the web but as part of another PR. This one alone focuses on adding the get started and guides content.